demo/app.py

```diff
@@ -0,0 +1,68 @@
+import json
+import re
+import requests
+from flask import Flask, request, render_template
+from requests.exceptions import ConnectionError
+
+
+app = Flask(__name__)
+app.config.from_pyfile("configuration.py")
+
+
+def init_session():
+    global session
+    session = requests.Session()
+    session.auth = (
+        app.config["AUTH_USERNAME"],
+        app.config["AUTH_PASSWORD"]
+    )
+    session.verify = False
+
```

Comment:
Just curious, why not using [flask-session](https://pythonhosted.org/Flask-Session/)?

Reply:
The session I'm using refers to the python requests session object. I only use it here over requests so that I can set my authentication once and not have to worry about it again. flask-session seems to be more for persisting user data over a user's session. I don't think I would be able to use flask-session in the same way I use python request's session here. Am I understanding this right?